resources/functionMap_php81delta.php

```diff
@@ -21,7 +21,12 @@
  */
 return [
 	'new' => [
-
+		'mysqli_fetch_field' => ['(stdClass&object{name: string, orgname: string, table: string, orgtable: string, def: string, db: string, catalog: "def", max_length: 0, length: int, charsetnr: string, flags: int, type: int, decimals: int})|false', 'result'=>'mysqli_result'],
```

Comment:
What's different about these on 8.1+?

Reply:
max_length is always zero starting from PHP 8.1, as documented in https://www.php.net/manual/en/mysqli-result.fetch-fields.php

Reply:
Here is the commit which changes it to be always 0: https://github.com/php/php-src/commit/890e4caf0be2bf625614e549f312ac18395651a0